app/assets/javascripts/app/app.js.erb

```diff
@@ -76,10 +76,10 @@
 // We turn automatic bootstrapping via %html{"ng-app" => "cloviFlowApp"}
 // so we can have AngularJS play nicely with Turbolinks.
 // See: http://stackoverflow.com/questions/14797935/using-angularjs-with-turbolinks
-$(document).on('ready page:load', function()
```

Comment:
I think it is better to just remove the code instead of commenting it out, no?

Reply:
Code removed. I think Turbolinks isn't used anywhere, but I still left the comment for the Stack Overflow link if needed.